app/views/wallet/TransactionView/SendMobPanel/SendMobForm.tsx

```diff
@@ -383,6 +384,13 @@ const SendMobForm: FC = () => {
           return a.alias.toUpperCase() > b.alias.toUpperCase() ? 1 : -1;
         });
 
+        const truncateContact = (contact: string, len: number) => {
```

Comment:
nit! try to use full meaning and words when naming things. In this can, you mean `length` instead of `len`. Abbreviations can become confusing (I'm probably guilty of this, I know I am for `err`)